conf/groups.conf.d/slurm.conf.example

```diff
@@ -3,19 +3,30 @@
 # Please see `man 5 groups.conf` for further details.
 #
 
+#
 # SLURM partition bindings
 #
-[slurm]
+[slurmpart,sp]
 map: sinfo -h -o "%N" -p $GROUP
 all: sinfo -h -o "%N"
 list: sinfo -h -o "%R"
 reverse: sinfo -h -N -o "%R" -n $NODE
 
+#
 # SLURM state bindings
 #
-[slurmstate]
-map: if [ "$GROUP" == "broken" ]; then sinfo -h -o "%N" -t down,drained; else sinfo -h -o "%N" -t $GROUP; fi
+[slurmstate,st]
+map: sinfo -h -o "%N" -t $GROUP
 all: sinfo -h -o "%N"
-list: sinfo -h -o "%T" | tr -d '~#$*'
-reverse: sinfo -h -N -o "%T" -n $NODE
-cache_time: 300
+list: sinfo -h -o "%T" | sed 's/[*$]//g'
+reverse: sinfo -h -N -o "%T" -n $NODE | sed 's/[*$]//g'
+cache_time: 60
+
+#
+# SLURM job bindings
+#
+[slurmjob,sj]
+map: squeue -j $GROUP -h -o "%N"
```

Comment:
if you refresh this patch, keep `-h` at the beginning of each command. It makes things simpler to understand.

Reply:
Good point.